### PR TITLE
AC Test Fix - LRAC-10420 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10420
https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1601539489

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsHidden.testcase [hunk 1]
```diff
@@ -267,8 +267,8 @@ definition {
 			ACDefinitions.viewUnhideEvent(customEventNameList = "${defaultEventName}");
 		}
 
-		task ("Hide default event") {
-			ACCustomEvents.hideCustomEvent(customEventName = "${defaultEventName}");
+		task ("Unhide all default events") {
+			ACDefinitions.hideAllEvents();
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsHidden.testcase:271]: hey @ViniLopes20 , aren't the task and action mixed up?
unhide... hide ?

in the next step of this test case you are doing the same task but with the unhideAllEvents action inside...

Could you please double-check?

Thanks!

Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsHidden.testcase:271]: Thanks for the explanation, @ViniLopes20 !

Looks good to me!

Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsHidden.testcase:271]: ci:forward

**Comment by alexcandidos** [comment]
Looks good to me!

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**